hack/make/binary

```diff
@@ -62,3 +62,18 @@ echo "Created binary: $DEST/$BINARY_FULLNAME"
 ln -sf "$BINARY_FULLNAME" "$DEST/docker$BINARY_EXTENSION"
 
 hash_files "$DEST/$BINARY_FULLNAME"
+
+# Add nested executables to bundle dir so we have complete set of
+# them available, but only if the native OS/ARCH is the same as the
+# OS/ARCH of the build target
+if [ "$(go env GOOS)/$(go env GOARCH)" == "$(go env GOHOSTOS)/$(go env GOHOSTARCH)" ]; then
+	echo "Copying nested executables into $DEST"
+	(set -x
+	if [ -x /usr/local/bin/runc ]; then
+		cp /usr/local/bin/runc $DEST/
```

Comment:
@jfrazelle it's also so that we can have these downloads available on master.dockerproject.org so that people can get everything they need to run docker.